code/modules/fishing/fish/_fish.dm

```diff
@@ -180,29 +206,30 @@
 	if(!istype(item, /obj/item/fish_feed))
 		return ..()
 	if(!item.reagents.total_volume)
-		balloon_alert(user, "[item] is empty!")
+		balloon_alert(user, "[item.name] is empty!")
 		return TRUE
 	if(status == FISH_DEAD)
-		balloon_alert(user, "[src] is dead!")
+		balloon_alert(user, "[name] [HAS_MIND_TRAIT(user, TRAIT_NAIVE) ? "isn't hungry" : "is dead!"]")
 		return TRUE
 	feed(item.reagents)
-	balloon_alert(user, "fed [src]")
+	balloon_alert(user, "fed [name]")
 	return TRUE
 
 /obj/item/fish/examine(mob/user)
 	. = ..()
 	if(HAS_MIND_TRAIT(user, TRAIT_EXAMINE_DEEPER_FISH))
 		if(status == FISH_DEAD)
-			. += span_deadsay("it's dead.")
-		var/list/warnings = list()
-		if(is_hungry())
-			warnings += "starving"
-		if(!HAS_TRAIT(src, TRAIT_FISH_STASIS) && !proper_environment())
-			warnings += "drowning"
-		if(health < initial(health) * 0.6)
-			warnings += "sick"
-			if(length(warnings))
-				. += span_warning("it's [english_list(warnings)]")
+			. += span_deadsay(HAS_MIND_TRAIT(user, TRAIT_NAIVE) ? "it's taking the big snooze" : "it's dead.")
```

Comment:
Between these two options, `"it's dead."` has a period, `"it's taking the big snooze"` doesn't
They're also not capitalized
```suggestion
			. += span_deadsay(HAS_MIND_TRAIT(user, TRAIT_NAIVE) ? "It's taking the big snooze." : "It's dead.")
```

Reply:
Ingame, this also doesn't seem to show up, and throws a runtime error instead:
```
runtime error:
type mismatch: "<span class=\'deadsay\'>" + 0
proc name: examine (/obj/item/fish/examine)
source file: code/modules/fishing/fish/_fish.dm,222
```